l10n_nl_xaf_auditfile_export/auditfile_export.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+##############################################################################
+#
+#    XAF Auditfile export
+#    Copyright (C) 2014 ONESTEiN BV (<http://www.onestein.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+ -->
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="auditfile_tree_view">
+            <field name="name">auditfile.tree</field>
+            <field name="model">auditfile.generate</field>
+            <field name="type">tree</field>
```

Comment:
'type' field is derived from the content of arch since 7.0. Specifying it explicitely is therefore deprecated and triggers warnings.